app/core/src/main/java/com/fsck/k9/helper/Utility.java

```diff
@@ -312,6 +328,95 @@ public static String extractMessageId(final String text) {
         return null;
     }
 
+    public static String getResizedImageFile(Context context, Uri uri, int circumference, int quality) {
```

Comment:
This functionality should be in its own class, `ImageResizer` or so. The methods also shouldn't be static, so the class can be injected as a dependency. Check out `AccountPreferenceSerializer` for an example where this works similarly.